deltablock.go

```
@@ -562,11 +588,18 @@ func performBackup(bsDriver BackupStoreDriver, config *DeltaBackupConfig, delta
 	destURL := config.DestURL
 	concurrentLimit := config.ConcurrentLimit
 
+	blockSize, err := config.getBackupBlockSize()
+	if err != nil {
+		logrus.WithError(err).Errorf("Failed to get backup block size, use default size %v instead", DEFAULT_BLOCK_SIZE)
```

```suggestion
		logrus.WithError(err).Warnf("Failed to get backup block size, use default size %v instead", DEFAULT_BLOCK_SIZE)
```

---

Do we really need the fallback?

---

For the backup creation, we can return the error directly. All backup creation request should contain the block size parameter after v1.10.